Microsoft.Toolkit.Uwp.Services/Services/Twitter/TwitterDataProvider.cs

```diff
@@ -636,12 +640,12 @@ private static string ExtractTokenFromResponse(string getResponse, TwitterOAuthT
 
             string getResponse;
 
-            using (var request = new HttpHelperRequest(new Uri(twitterUrl), Windows.Web.Http.HttpMethod.Get))
+            using (var request = new HttpRequestMessage(HttpMethod.Get, new Uri(twitterUrl)))
             {
-                using (var response = await HttpHelper.Instance.SendRequestAsync(request).ConfigureAwait(false))
+                using (var response = await client.SendAsync(request).ConfigureAwait(false))
                 {
-                    var data = await response.GetTextResultAsync().ConfigureAwait(false);
-                    if (response.Success)
+                    var data = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
```

Comment:
There is an encoding issue here when testing the Twitter service, data is unreadable.